scripts/config.py

```diff
@@ -595,51 +596,74 @@ def set(self, name, value=None):
             raise ValueError(f'Feature is unstable: \'{name}\'')
 
         if name not in self.settings:
-            self.crypto_config.templates.append((name, '', '#define ' + name + ' ' + '1'))
+            self.configfile.templates.append((name, '', '#define ' + name + ' '))
```

Comment:
The templates shouldn't contains the value. It can be modified.
The value is stored in `Config.settings` and will printed by the `ConfigFile._format_template`.  See [here](https://github.com/Mbed-TLS/mbedtls/blob/db543271b568a594c98ff24c7e5b1e3ab8c0e6be/scripts/config.py#L498).

Reply:
`PSA_WANT_xxx` is [specified](https://github.com/Mbed-TLS/mbedtls/blob/mbedtls-3.6.0/docs/proposed/psa-conditional-inclusion-c.md#configuration-symbol-syntax) as 1 for enabled and unset for disabled. My original intent was that 0 would be disabled, but we ended up with a lot of `#if defined(PSA_WANT_xxx)` rather than `#if PSA_WANT_xxx` and in 2.x and 3.x we standardized on `defined()`. I'd like to revisit this in 4.0 though.

Reply:
Ah, ok, Sorry - I was misreading the code as you having removed the value from the previous version.